README-Polish

mbiesiad opened the pull request:
Polish translation

mbiesiad commented:
Hi there! Really great project. Here is translation into Polish, finished: https://github.com/mbiesiad/nodebestpractices
Best regards

goldbergyoni commented:
@mbiesiad Wow, amazing work! 🏅 😲 

Just to ensure I get the status straight: By now, the 1st page is translated but not the inner pages? Do they refer to the English version?

mbiesiad commented:
My pleasure! :-) yea, now I see. Yup, at this moment that's refered to the English version. Thank you so much for your note, please give me a time, I will let you know when it will be done. [WIP] Stay tuned!

mbiesiad commented:
@goldbergyoni uff, done. Looks good for me, I hope there will not be any bigger fixes. Added & translated other subsites. I hope you enjoy! Greetings

kekuu commented on .operations/writing-guidelines.polish.md:
```diff
@@ -0,0 +1,31 @@
+
+Jak zwiększamy komfort czytania i uczenia się dla naszych gości.
+
+## 1. Proste jest lepsze niż lepsze
```
I think there should be simply: 'Proste jest lepsze'

mbiesiad replied:
agree, fixed. done, thanks a lot!

kekuu commented on .operations/writing-guidelines.polish.md:
```diff
@@ -0,0 +1,31 @@
+
+## 4. Spójne formatowanie
+
+Treść jest prezentowana przy użyciu stałych szablonów. Wszelkie przyszłe treści muszą być zgodne z tym samym szablonem. Jeśli chcesz dodać nowe punktory, skopiuj format punktora z istniejącego i wypuść go do swoich potrzeb. Aby uzyskać dodatkowe informacje, zobacz [ten szablon](https://github.com/i0natan/nodebestpractices/blob/master/sections/template.md).
```
' i wypuść go do ' should be " i rozszerz go do "

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on .operations/writing-guidelines.polish.md:
```diff
@@ -0,0 +1,31 @@
+
+## 5. To na temat Node.js
+
+Każda rada powinna dotyczyć bezpośrednio Node.js, a nie ogólnie oprogramowania. Kiedy radzimy zaimplementować ogólny wzorzec / regułę w Node.js, treść powinna koncentrować się na implementacji Node. Na przykład, gdy ze względów bezpieczeństwa zalecamy sanitize danych wejściowych z żądań, należy użyć Node-lingo - „Użyj oprogramowania pośredniego do sanitize danych wejściowych”. Jeśli element nie m ...
```
"sanitize danych wejściowych z żądań " change to: " ochronę danych wrażliwych z wejściowych żądań"

kekuu replied:
Same: „Użyj oprogramowania pośredniego do sanitize danych wejściowych” change to: „Użyj oprogramowania pośredniego do oczyszczenia danych wejściowych"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on .operations/writing-guidelines.polish.md:
```diff
@@ -0,0 +1,31 @@
+
+## 6. Tylko wiodący dostawcy
+
+Czasami przydatne jest podanie nazw dostawców, którzy mogą rozwiązać niektóre wyzwania i problemy, takie jak pakiety npm, narzędzia open source, a nawet produkty komercyjne. Aby uniknąć przytłaczających długich list lub rekomendować projekty niestabilne i niestabilne, opracowaliśmy następujące zasady:
```
"niestabilne i niestabilne" change to: "nierenomowane i niestabilne"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on .operations/writing-guidelines.polish.md:
```diff
@@ -0,0 +1,31 @@
+Czasami przydatne jest podanie nazw dostawców, którzy mogą rozwiązać niektóre wyzwania i problemy, takie jak pakiety npm, narzędzia open source, a nawet produkty komercyjne. Aby uniknąć przytłaczających długich list lub rekomendować projekty niestabilne i niestabilne, opracowaliśmy następujące zasady:
+
+- Polecamy tylko 3 najlepszych dostawców - dostawcę, który pojawia się w 3 najlepszych wynikach wyszukiwania (Google lub GitHub posortowane według popularności) dla danego odpowiedniego słowa kluczowego, możemy uwzględnić w naszej rekomendacji.
+- Jeśli jest to pakiet npm, należy go również pobrać średnio co najmniej 750 razy dziennie.
```
"Jeśli jest to pakiet npm, należy go również pobrać średnio co najmniej 750 razy dziennie." change to:  "musi być pobierany średnio co najmniej 750 razy dziennie."

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on sections/errorhandling/catchunhandledpromiserejection.polish.md:
```diff
@@ -0,0 +1,87 @@
+
+### Wyjaśnienie jednego akapitu
+
+Zazwyczaj większość współczesnego kodu aplikacji Node.js / Express działa w ramach obietnic - czy to w .then handler, wywołaniu zwrotnym funkcji, czy w bloku catch. Zaskakujące jest to, że jeśli programista nie pamięta o dodaniu klauzuli .catch, błędy zgłaszane w tych miejscach nie są obsługiwane przez moduł obsługi zdarzeń uncaughtException i znikają. Najnowsze wersje Node dodały komunikat ostrz ...
```
"że każdy błąd promise " change to: "że każdy błąd obietnicy" to be more consistent

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on sections/errorhandling/centralizedhandling.polish.md:
```diff
@@ -0,0 +1,164 @@
+
+### Wyjaśnienie jednego akapitu
+
+Bez jednego obiektu dedykowanego do obsługi błędów większe są szanse na poważne błędy ukryte pod radarem z powodu niewłaściwej obsługi. Obiekt obsługi błędów jest odpowiedzialny za uwidocznienie błędu, na przykład poprzez napisanie do dobrze sformatowanego programu rejestrującego, wysyłanie zdarzeń do jakiegoś produktu monitorującego, takiego jak [Sentry](https://sentry.io/), [Rollbar](https://ro ...
```
"-> nazywa się scentralizowany moduł obsługi błędów ->" just: "-> scentralizowany moduł obsługi błędów ->"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on sections/errorhandling/documentingusingswagger.polish.md:
```diff
@@ -0,0 +1,52 @@
+
+### Wyjaśnienie jednego akapitu
+
+REST API zwracają wyniki przy użyciu kodów stanu HTTP, użytkownik API musi bezwzględnie wiedzieć nie tylko o schemacie interfejsu API, ale także o potencjalnych błędach - osoba wywołująca może wtedy złapać błąd i taktownie go obsłużyć. Na przykład dokumentacja interfejsu API może z góry stwierdzać, że status HTTP 409 jest zwracany, gdy nazwa klienta już istnieje (zakładając, że interfejs API reje ...
```
Forgotten dot at the end.

kekuu commented on sections/errorhandling/operationalvsprogrammererror.polish.md:
```diff
@@ -0,0 +1,85 @@
+
+### Wyjaśnienie jednego akapitu
+
+Rozróżnienie następujących dwóch typów błędów zminimalizuje przestoje aplikacji i pomoże uniknąć szalonych błędów: Błędy operacyjne odnoszą się do sytuacji, w których rozumiesz, co się stało i jego wpływ - na przykład zapytanie do jakiejś usługi HTTP nie powiodło się z powodu problemu z połączeniem. Z drugiej strony błędy programisty odnoszą się do przypadków, w których nie masz pojęcia, dlaczego ...
```
"i jego wpływ" change to: "i ich wpływ"

kekuu replied:
" która przecieka pamięć" should be: " z której wycieka pamięć"

kekuu replied:
Last sentence: "Gdy pojawia się błąd programisty, rzeczy stają się zagmatwane, aplikacja może być niespójna i nie ma nic lepszego niż ją po prostu zrestartować"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on sections/errorhandling/shuttingtheprocess.polish.md:
```diff
@@ -0,0 +1,99 @@
+# Opuść ten proces z wdziękiem, gdy do miasta przybywa nieznajomy
+
+### Wyjaśnienie jednego akapitu
```
"### Wyjaśnienie jednego akapitu" maybe change it to: "Wyjaśnienie w jednym akapicie"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on sections/errorhandling/shuttingtheprocess.polish.md:
```diff
@@ -0,0 +1,99 @@
+
+### Wyjaśnienie jednego akapitu
+
+Gdzieś w twoim kodzie obiekt procedury obsługi błędów jest odpowiedzialny za podjęcie decyzji, jak postępować, gdy błąd zostanie zgłoszony - jeśli błąd jest zaufany (tj. błąd operacyjny, zobacz dalsze wyjaśnienie w najlepszych praktykach #3), zapisywanie w pliku dziennika może być wystarczające. Sprawa staje się pokręcona, jeśli błąd nie jest znany - oznacza to, że niektóre komponenty mogą być wa ...
```
"Na przykład, zakładając singletonową, stateful token issuer service, która zgłosiła wyjątek i utraciła swój stan" change to: "Na przykład, zakładając singletonowy serwis stateful token issuer, który zgłosił wyjątek i utracił swój stan"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

kekuu commented on sections/production/apmproducts.polish.md:
```diff
@@ -0,0 +1,25 @@
+
+### Wyjaśnienie jednego akapitu
+
+APM (application performance monitoring) odnosi się do rodziny produktów, która ma na celu monitorowanie wydajności aplikacji od początku do końca, również z perspektywy klienta. Podczas gdy tradycyjne rozwiązania monitorujące koncentrują się na wyjątkach i niezależnych pomiarach technicznych (np. śledzeniu błędów, powolnych punktach końcowych serwera itp.), w prawdziwym świecie nasza aplikacja m ...
```
"od końca do końca," should be original: "end-to-end" or "na całej drodze"

kekuu commented on sections/production/bestateless.polish.md:
```diff
@@ -0,0 +1,42 @@
+# Bądź stateless, zabijaj Serwery prawie codziennie
```
"Serwery " change to: "serwery "

kekuu commented on sections/production/frontendout.polish.md:
```diff
@@ -0,0 +1,45 @@
+
+### Wyjaśnienie jednego akapitu
+
+W klasycznej aplikacji internetowej backend obsługuje frontend / grafikę w przeglądarce, bardzo powszechnym podejściem w świecie Node jest użycie Express'owego oprogramowania pośredniego do usprawnienia plików statycznych w kliencie. ALE - Node nie jest typową aplikacją internetową, ponieważ wykorzystuje pojedynczy wątek, który nie jest zoptymalizowany do obsługi wielu plików jednocześnie. Zamias ...
```
"odwrotnego proxy" should be: "reverse-proxy"

kekuu commented on sections/production/delegatetoproxy.polish.md:
```diff
@@ -0,0 +1,51 @@
+# Deleguj wszystko, co możliwe (np. zawartość statyczną, gzip) do odwrotnego proxy
```
"odwrotnego proxy" should be: "reverse-proxy"

kekuu commented on sections/production/frontendout.polish.md:
```diff
@@ -0,0 +1,45 @@
+
+Twoje optymalne rozwiązanie może mieć jedną z następujących postaci:
+
+1. Korzystanie z odwrotnego proxy - twoje pliki statyczne będą znajdować się tuż obok aplikacji Node, tylko żądania do folderu plików statycznych będą obsługiwane przez proxy, które znajduje się przed aplikacją Node, taką jak nginx. Dzięki takiemu podejściu aplikacja Node jest odpowiedzialna za wdrażanie plików statycznych, ale nie za ich obsługę. Twój kolega z interfejsu użytkownika pokocha to p ...
```
"odwrotnego proxy" should be: "reverse-proxy"

kekuu commented on sections/production/guardprocess.polish.md:
```diff
@@ -0,0 +1,17 @@
+
+### Wyjaśnienie jednego akapitu
+
+Na poziomie podstawowym procesy węzłów muszą być chronione i restartowane w przypadku awarii. Mówiąc prosto, dla małych aplikacji i tych, którzy nie używają kontenerów - narzędzia takie jak [PM2](https://www.npmjs.com/package/pm2-docker) są idealne, ponieważ zapewniają prostotę, możliwości ponownego uruchamiania oraz bogatą integrację z węzłem. Inni z dużymi umiejętnościami w Linuksie mogą używać ...
```
"pojemniki" -> "kontenery"
Missing dot at the end.

kekuu commented on sections/production/lockdependencies.polish.md:
```diff
@@ -0,0 +1,69 @@
+
+### Wyjaśnienie jednego akapitu
+
+Twój kod zależy od wielu zewnętrznych pakietów, powiedzmy, że „wymaga” i użyj momentjs-2.1.4, a następnie domyślnie po wdrożeniu do produkcji npm może pobrać momentjs 2.1.5, co niestety wprowadza kilka nowych błędów do tabeli. Użycie plików konfiguracyjnych npm i argumentu ``–save-exact = true`` instruuje npm, aby odwoływał się do *dokładnie* tej samej wersji, która została zainstalowana, więc na ...
```
"i użyj momentjs-2.1.4" should be: "i używa momentjs-2.1.4,"
"więc następnym razem uruchomisz" should be: "więc gdy następnym razem uruchomisz"

mbiesiad replied:
Agree, fixed. Done, thanks a lot!

mbiesiad commented:
@goldbergyoni updated, done. After CR. Enjoy! :-)

goldbergyoni commented:
@mbiesiad  This is amazing work! ❤️ ❤️ ❤️ 

We will include this is the news section with your name & Tweet about it. Well, that's all we can do but maybe you can also share in local groups?

What's your Twitter handle?

We will soon also create a specific section for translators credit, for now adding you to the global credits list, I assume you also put your name in the translations section?

@all-contributors please add @mbiesiad  for content

goldbergyoni commented:
@mbiesiad Oh, as I clicked 'Merge' I faced some conflict, can you help in resolving this?

mbiesiad commented:
> 
> 
> @mbiesiad Oh, as I clicked 'Merge' I faced some conflict, can you help in resolving this?

Conflict info:
![merge-conflict](https://user-images.githubusercontent.com/18367606/77925191-dc08f900-72a4-11ea-8a5a-c604983cd51f.png)

Sure, I have added Polish flag etc to the README.md previously, sorry my fault. Now, I deleted this now to avoid merge conflict. Now should be OK, please let me know, if is.
That's a snippet for paste if you want to add later:
`Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chinese.md), [![BR](/assets/flags/BR.png)**BR**](/README.brazilian-portuguese.md), [![RU](/assets/flags/RU.png)**RU**](/README.russian.md), [![PL](/assets/flags/PL.png)**PL**](/README.polish.md), [(![ES](/assets/flags/ES.png)**ES**, ![FR](/assets/flags/FR.png)**FR**, ![HE](/assets/flags/HE.png)**HE**, ![KR](/assets/flags/KR.png)**KR** and ![TR](/assets/flags/TR.png)**TR** in progress!)](#translations)`

mbiesiad commented:
Sounds great! Thank you! Sure, good idea, I will.
My Twitter handle: [@michalbiesiada](https://twitter.com/michalbiesiada)
Ok, sounds really nice! Do you mean this section? ("readme -> Translations - Completed translations").

goldbergyoni commented:
@mbiesiad

Merged! 🍾🥂🍾🥂

Add yourself to the top of the Polish home page, and also to the path you referred to (finished translations)

I'm adding you to contributors and Tweeting about this. Cool?

goldbergyoni commented:
@all-contributors please add @mbiesiad for content

mbiesiad commented:
> 
> 
> @mbiesiad
> 
> Merged! 🍾🥂🍾🥂
> 
> Add yourself to the top of the Polish home page, and also to the path you referred to (finished translations)
> 
> I'm adding you to contributors, putting in news and Tweeting about this. Cool?

@goldbergyoni Sure, thank you! I really appreciate it. 🔥  Ok, sounds great! 🥇
